imports/ui/components/error.html

```diff
@@ -1,91 +1,53 @@
-        <div class="container">
-            <div class="row title-row">
-                <div class="col-12 landing2-title">This is what OGV does best</div>
-            </div>
-            <div class="row">
-                <div class="col-12 col-lg-3 col-sm-6">
-                    <p><img class="mobile-view-fileUpload" src="/icons/fileUpload.png" width="200px" height="200px" /></p>
-                    <h3 class="landing2-titles">Upload Model</h3>
-                    <p class="landing2-text">Countless number of .g files can be uploaded from your browsers.</p>
-                </div>
-                <div class="col-12 col-lg-3 col-sm-6">
-                    <p><img class="mobile-view-computer" src="/icons/computer.png" width="200px" height="200px" /></p>
-                    <h3 class="landing2-titles">Visualize</h3>
-                    <p class="landing2-text">Visualize your models in three dimensions and communicate more effectively.</p>
-                </div>
-                <div class="col-12 col-lg-3 col-sm-6">
-                    <p><img class="mobile-view-share" src="/icons/share.png" width="200px" height="200px" /></p>
-                    <h3 class="landing2-titles">Embed and Share</h3>
-                    <p class="landing2-text">Embed and share your models anywhere on social sites, blogs or others.</p>
-                </div>
-                <div class="col-12 col-lg-3 col-sm-6">
-                    <p><img class="mobile-view-download" src="/icons/download-black.png" width="200px" height="200px" /></p>
-                    <h3 class="landing2-titles">Download</h3>
-                    <p class="landing2-text">Fetch, download and use the 3D models for printing and more.</p>
-                </div>
-            </div>
-        </div>
-    </section>
-    <!-- Slide 2 over-->
+			<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="762px"
+			 height="331px" viewBox="0 0 762 331" enable-background="new 0 0 762 331" xml:space="preserve" class="cloud slower">
+				<path fill="#333" d="M715.394,228h-16.595c0.79-5.219,1.201-10.562,1.201-16c0-58.542-47.458-106-106-106
+        c-8.198,0-16.178,0.932-23.841,2.693C548.279,45.434,488.199,0,417.5,0c-84.827,0-154.374,65.401-160.98,148.529 C245.15,143.684,232.639,141,219.5,141c-49.667,0-90.381,38.315-94.204,87H46.607C20.866,228,0,251.058,0,279.5
+        S20.866,331,46.607,331h668.787C741.133,331,762,307.942,762,279.5S741.133,228,715.394,228z" />
+			</svg>
 
-    <!-- Footer Start -->
-    <footer class="footer" id="footer">
-        <div class="container">
-            <p class="text-center">All trademarks referenced herein are the properties of their respective owners. This site is not sponsored, endorsed,
-                or run by the U.S. Government.</p>
-        </div>
-    </footer>
-    <!-- Footer End -->
-</template>
\ No newline at end of file
+			<div class="container mainContent" id="mainContent">
```

Comment:
Probably where the problem is coming from is the way this container and frame is set up